packaging/rpm_chrootbuild

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
```

Comment:
This file seems to be DAOS only environment as well.

Reply:
Sorry, wrong repo, meant for the forked one